pkg/watch/mux.go

```diff
@@ -162,6 +165,7 @@ func (m *Broadcaster) loop() {
 		}
 		m.distribute(event)
 	}
+	m.distributing.Done()
```

Comment:
No-- a channel close doesn't blow away all the data in the channel, it just prevents further writes.